docs/cvl/invariants.md

```diff
@@ -37,8 +37,11 @@ preserved_block ::= "preserved"
                     [ "with" "(" params ")" ]
                     block
 
-method_signature ::= id "(" [ evm_type [ id ] { "," evm_type [ id ] } ] ")"
+method_signature ::= [ contract_name "." ] id  "(" [ evm_type [ id ] { "," evm_type [ id ] } ] ")"
                      | "fallback" "(" ")"
+
+contract_name ::= id
+                | "*"
```

Comment:
isn't this supposed to be `_`?

Reply:
After looking at the code, you are correct; it should be `_`.  For some reason, I was under the impression that the character was `*`.  Fixed.